src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -2647,9 +2632,9 @@ namespace HeatBalFiniteDiffManager {
         updatedThermalConductivity = materialDefinition->phaseChange->getConductivity(temperatureUpdated);
     }
 
-    bool findAnySurfacesUsingConstructionAndCondFD(EnergyPlusData &state, int const constructionNum)
+    bool findAnySurfacesUsingConstructionAndCondFD(EnergyPlusData const &state, int const constructionNum)
```

Comment:
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:2650]:(style),[constParameter],Parameter 'state' can be declared as reference to const
[src/EnergyPlus/HeatBalFiniteDiffManager.cc:2652]:(style),[constVariable],Variable 'thisSurface' can be declared as reference to const